tests/acceptance/10_files/02_maintain/absolute_link_and_no_move_obstructions.cf

```diff
@@ -44,8 +44,8 @@ bundle agent test
 
   files:
       "$(G.testfile)"
-      classes => init_if_failed("test_ok"),
-      link_from => test_link;
+          classes => init_if_failed("test_ok"),
+          link_from => test_link;
```

Comment:
`reindent.pl` just runs Emacs to reindent the code, so you need to look at the Emacs indentation settings (see the script, they are in `contrib/cfengine-code-style.el` for C and CFE code). It should Just Work.

We used to indent attributes 0 spaces, that's why older code has that (it should all be reindented some day).